indexer/ai/src/main/java/com/google/cloudsearch/ai/AISkillSet.java

```diff
@@ -0,0 +1,66 @@
+package com.google.cloudsearch.ai;
+
+import com.sun.org.apache.bcel.internal.generic.ANEWARRAY;
+import org.checkerframework.checker.units.qual.C;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class AISkillSet {
+
+    private List<AISkill> skillSet;
+    public void setSkillSet(List<AISkill> skillSet){
```

Comment:
we don't need set and get for the list of AISkill

Reply:
Actually as I have mentioned in local repository, I need to get the list of all skills from the AISkillSet object and then for each skill I am calling execute(), That's why I have kept the get and set for this.